release/sil/sil.glk-arab.ptwl1/sil.glk-arab.ptwl1.kpj

```diff
@@ -19,12 +19,12 @@
       <ID>id_f008039edc7c2114f0c7b27dfdb93884</ID>
       <Filename>sil.glk-arab.ptwl1.model.kps</Filename>
       <Filepath>source\sil.glk-arab.ptwl1.model.kps</Filepath>
-      <FileVersion>1.0.1</FileVersion>
+      <FileVersion>1.0.2</FileVersion>
```

Comment:
This file looks like it belongs on a separate PR

Reply:
One way to undo this change:
1. check out your `sil.bcc-arab` branch
2. edit this file back to 1.0.1
3. commit and push the branch.